src/Shared/Difficulty.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Birds.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Birds.Shared
+{
+    public enum Difficulty
```

Comment:
Rename this enum to `Difficulties`. By default, enums must be in plural

Reply:
Done!